econsa/correlation.py

```diff
@@ -10,7 +10,7 @@
 from statsmodels.stats.correlation_tools import corr_nearest
 
 
-def gc_correlation(marginals, corr, force_calc=False, rule="sobol", num_points=1000):
+def gc_correlation(marginals, corr, force_calc=False, order=15):
     r"""Correlation for Gaussian copula.
```

Comment:
Please add to docstring that we use Gauss Hermite quadrature to implement the algorithm .